contribution.md

```diff
@@ -0,0 +1,38 @@
+# How to contribute.
+
+## Required tools
+
+- node.js
+- npm
+- on mac-os, you have to install xcode command line developer tools (run xcode-select --install)
```

Comment:
You're probably right but I'm curious what is required there?

Reply:
Sorry. I dont really remember why I had to do that. (The only thing I remember is that it was painfull to get a working configuration on my computer ;-) ).